ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/validator/preexecution/TimeUpdatesProvider.scala

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.validator.preexecution
+
+import com.daml.lf.data.Time.Timestamp
+
+object TimeUpdatesProvider {
+  private[this] val NowFakeTimeUpdatesProvider: TimeUpdatesProvider = () => Some(Timestamp.now())
+
+  // TODO replace with actual time update log entry read
```

Comment:
Please remove TODO (and possibly track this task in the backlog).

Reply:
Done.